WordPress/wordcamp.org
# Blocks: Match speakers output to mockups

Style tweaks and some minor component cleanup.

Fixes #94.

![Screen Shot 2019-07-26 at 1 43 13 PM](https://user-images.githubusercontent.com/541093/61970737-63ca9500-afab-11e9-9a1a-b7ea3beca2fd.png)

To test:

1. Add a Speakers block to a post.
2. Toggle the content settings on & off.
3. Display should match the screenshot (given some flexibility for theme styles).
